.travis_setup.sh

```diff
@@ -2,6 +2,12 @@
 
 set -eux
 
+# Install the default used DB if DB is not set.
+if [[ -z ${DB-} ]]; then
+  sudo apt-get update -qq
+  sudo apt-get install -qq mysql-server-5.7 mysql-client-core-5.7 mysql-client-5.7
```

Comment:
I see this line causes an install error on the current Travis CI case on Ubuntu trusty.
https://travis-ci.org/github/brianmario/mysql2/jobs/759562170#L1200

If we still like to keep the Travis trusty case, maybe if changing `mysql-server-5.7 mysql-client-core-5.7 mysql-client-5.7` to `mysql-server mysql-client-core mysql-client`, we might avoid the error. I wanted to add the version string of the mysql as much as possible to avoid installing the unintended version.

Reply:
Rebased. The Github Actions log is [here](https://github.com/junaruga/mysql2/actions/runs/580986457).